app/assets/stylesheets/main.css.scss

```diff
@@ -0,0 +1,44 @@
+#container {
+  margin: 48px auto;
+  width: 960px;
+
+  > {
+    section {
+      clear: both;
+    }
+
+    header {
+      margin-bottom: 46px;
+
+      > h1 {
+        float: left;
+        margin: 0;
+      }
+    }
+  }
+}
+
+.navigation-links {
+  float: right;
+
+  li {
+    display: inline-block;
+    margin: 0 16px;
+  }
+}
+
+.clearfix:after {
+  content: ".";
```

Comment:
Properties should be ordered clear, content, display, height, line-height, visibility